notebooks/spark/AerospikeSparkPython.ipynb

```diff
@@ -5,7 +5,7 @@
    "metadata": {},
    "source": [
     "# Aerospike Connect for Spark Tutorial for Python\n",
```

Comment:
General comments: Remove all unused dead code from this notebook. It will make it look cleaner.